### PR TITLE
Drop dependency on doctrine/common

### PR DESCRIPTION
**Pull request by Majkl578**
Common is no more. \o/

### CODE REVIEW

File: lib/Doctrine/ORM/Tools/Console/Command/RunDqlCommand.php [hunk 1]
```diff
@@ -101,6 +95,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $resultSet = $query->execute([], constant($hydrationMode));
 
-        $ui->text(Debug::dump($resultSet, $input->getOption('depth'), true, false));
+        $dumper = new CliDumper(static function (string $payload) use ($output) : void {
+            $output->write($payload);
+        });
+        $dumper->dump((new VarCloner())->cloneVar($resultSet));
```
Comment (Ocramius) [inline lib/Doctrine/ORM/Tools/Console/Command/RunDqlCommand.php]: This change specifically requires an integration test to be added

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: lib/Doctrine/ORM/Tools/Console/Command/RunDqlCommand.php (hunk 1 above)
Comment (Majkl578) [reply lib/Doctrine/ORM/Tools/Console/Command/RunDqlCommand.php]: There already is a test, which was failing without this change:
https://github.com/doctrine/doctrine2/blob/master/tests/Doctrine/Tests/ORM/Tools/Console/Command/RunDqlCommandTest.php#L67

**Comment by Majkl578** [comment]
Scrutinizer pointed out one more occurence of ClassLoader in `bin/doctrine-pear.php`, but afaik we don't use PEAR for ages?

**Review by Ocramius — APPROVED** [review]
:ship:

**Pull request merged**